app/views/layouts/application.html.erb

```diff
@@ -3,8 +3,6 @@
 <head>
   <title><%= yield :title %> - GOV.UK</title>
   <!--[if gt IE 8]><!--><%= stylesheet_link_tag "application" %><!--<![endif]-->
-  <!--[if IE 6]><%= stylesheet_link_tag "application-ie6" %><script>var ieVersion = 6;</script><![endif]-->
-  <!--[if IE 7]><%= stylesheet_link_tag "application-ie7" %><script>var ieVersion = 7;</script><![endif]-->
```

Comment:
These scripts also need to be removed from https://github.com/alphagov/manuals-frontend/blob/master/config/initializers/assets.rb

Reply:
oh didn't think to look there as well. thanks